lib/cdo/script_constants.rb

```diff
@@ -91,6 +91,7 @@ module ScriptConstants
       COURSE2_NAME = 'course2'.freeze,
       COURSE3_NAME = 'course3'.freeze,
       COURSE4_NAME = 'course4'.freeze,
+      COURSE_CSL_NAME = 'csl-vn'.freeze
```

Comment:
I believe we use this category for more things that just deciding whether or not to translate something; unless this course is actually a part of our officially-supported content, it probably shouldn't go here.

Reply:
+1. It should either be in `ADDITIONAL_I18N_SCRIPTS` or just called out specifically in `i18n?`.